FirebaseSessions/Tests/TestApp/Shared/AppQualityDevAppApp.swift

```diff
@@ -15,10 +15,13 @@
 
 import SwiftUI
 import FirebaseCore
+import FirebaseSessions
 
 @main
-struct AppQualityDevAppApp: App {
-  init() {
+class AppQualityDevAppApp: App {
+  required init() {
```

Comment:
I remember when talking about how session dependency works we said the dependencies would be added when loaded to memory, like `+ load()`, so is `required init()` equivalent to `+ load()`?

Reply:
So I'm cheating a little bit. Swift doesn't have the same `load` function (at least I couldn't figure it out in 5 mins of Googling), so I just call this function at the start up the app, but before the Firebase SDKs start up.

So technically this solution is called later than `load`, but it's earlier than Firebase Initialization `FirebaseApp.configure()`, so it's ok.

Reply:
Got it, makes sense!